sdk/communication/azure-communication-callingserver/tests/test_call_connection.py

```diff
@@ -0,0 +1,1070 @@
+
+        call_connection = _mock_utils.create_mock_call_connection(
+            call_connection_id,
+            status_code=200,
+            payload=_test_constants.GetCallResponsePayload,
+            use_managed_identity=use_managed_identity
+            )
+
+        result = call_connection.get_call()
+        CallConnectionUnitTestUtils.verify_get_call_result(result)
+
+    @parameterized.expand(CallConnectionUnitTestUtils.data_source_test_get_call())
+    def test_get_call_failed(
+        self,
+        test_name, # type: str
+        call_connection_id, # type: str
+        use_managed_identity = False, # type: bool
+        ):
+
+        call_connection = _mock_utils.create_mock_call_connection(
+            call_connection_id,
+            status_code=404,
+            payload=_test_constants.ErrorPayload,
+            use_managed_identity = use_managed_identity
+            )
+
+        raised = False
+        try:
+            call_connection.get_call()
+        except:
+            raised = True
+        assert raised == True
+
+    @parameterized.expand(CallConnectionUnitTestUtils.data_source_test_delete())
+    def test_delete_succeed(
+        self,
+        test_name, # type: str
+        call_connection_id, # type: str
+        use_managed_identity = False, # type: bool
+        ):
+
+        call_connection = _mock_utils.create_mock_call_connection(
+            call_connection_id,
+            status_code=202,
+            payload=_test_constants.GetCallResponsePayload,
+            use_managed_identity=use_managed_identity
+            )
+
+        call_connection.delete()
+        assert call_connection.call_connection_id == _test_constants.CALL_ID
+
+    @parameterized.expand(CallConnectionUnitTestUtils.data_source_test_delete())
+    def test_delete_failed(
+        self,
+        test_name, # type: str
+        call_connection_id, # type: str
+        use_managed_identity = False, # type: bool
+        ):
+
+        call_connection = _mock_utils.create_mock_call_connection(
+            call_connection_id,
+            status_code=404,
+            payload=_test_constants.ErrorPayload,
+            use_managed_identity = use_managed_identity
+            )
+
+        raised = False
+        try:
+            call_connection.delete()
+        except:
+            raised = True
+        assert raised == True
+
+    @parameterized.expand(CallConnectionUnitTestUtils.data_source_test_hang_up())
+    def test_hang_up_succeed(
+        self,
+        test_name, # type: str
+        call_connection_id, # type: str
+        use_managed_identity = False, # type: bool
+        ):
+
+        call_connection = _mock_utils.create_mock_call_connection(
+            call_connection_id,
+            status_code=202,
+            payload=None,
+            use_managed_identity=use_managed_identity
+            )
+
+        call_connection.hang_up()
+        assert call_connection.call_connection_id == _test_constants.CALL_ID
```

Comment:
Same as Delete

Reply:
There are a few more places where this happens. I'm not going to call them all out. If it is an issue, please fix all of them.